frontend/src/pages/RegistrationPage.js

```diff
@@ -67,66 +68,99 @@ class RegistrationPage extends Component{
-                    <p><font size ="5" color="#1DE28F"> Sign Up </font></p>
-                    <form
-                        onSubmit={this.onSubmit}
-                    >
-                        <p>
-                        <input
-                            type="text"
-                            className="text-line"
-                            name="username"
-                            placeholder="Username"
-                            size="12"
-                            value={this.state.userField}
-                            onChange={this.onUserChange}
-                        />
-                        </p>
-
-                        <p>
+                <body className="align">
+                  <div className="grid">
+                    <form>
+                        <div className="form__field">
+                            {this.state.redirect && <Redirect to={this.state.redirect} />}
+                            <form onSubmit={this.onSubmit}/>
+                            <label>
+                                <img
+                                    src={`http://${process.env.REACT_APP_IP}/res/username.png`}
+                                    alt="username"
+                                    onClick={this.onLogoClick}
+                                />
+                            </label>
+                            <input
+                                id="register__username"
+                                type="username"
+                                name="username"
+                                className="form__input"
+                                placeholder="Username"
+                                value={this.state.userField}
+                                onChange={this.onUserChange}
+                                required>
+                          </input>
+                      </div>
+                      <div className="form__field">
+                          <label for="registerpassword1">
+                              <img
+                                  src={`http://${process.env.REACT_APP_IP}/res/password.png`}
+                                  alt="password"
+                                  onClick={this.onLogoClick}
+                              />
+                             </label>
                         <input
+                            id="password1"
                             type="password"
-                            className="text-line"
-                            name="password"
+                            className="form__input"
+                            name="password1"
                             placeholder="Password"
-                            size="12"
-                            value={this.state.passField}
+                            value={this.state.passField1}
                             onChange={this.onPassChange}
                             onKeyDown={this.onEnterPress}
```

Comment:
Remove this line as it will submit before the user has the chance to fill in the last field